samples/excel/tables/apply-values-filter.yaml

```diff
@@ -0,0 +1,137 @@
+                await context.sync();
+            }
+            catch (error) {
+                OfficeHelpers.Utilities.log(error);
+            }
+        })
+    }
+
+
+    /** Filter the data in the table*/
+    function filterData() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to get the sheet
+                sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                // Call the helper to filter the table
+                await queueFilterData(sheet);
+
+                // Run all of the queued up commands
+                await context.sync();
+            }
+            catch (error) {
+                OfficeHelpers.Utilities.log(error);
+            }
+        })
+    }
+
+    // Create the table
+    async function queueCreateTable(sheet: Excel.Worksheet) {
+
+            // Queue a command to add a table to the sheet
+            let expensesTable = sheet.tables.add('A1:D1', true);
+            expensesTable.name = "ExpensesTable";
+
+            // Queue a command to set the header row
+            expensesTable.getHeaderRowRange().values = [["DATE", "MERCHANT", "CATEGORY", "AMOUNT"]];
+
+            // Queue a command to add rows to the table
+            expensesTable.rows.add(null, [
+                ["1/1/2017", "THE PHONE COMPANY", "Communications", "$120"],
+                ["1/2/2017", "NORTHWIND ELECTRIC CARS", "Transportation", "$142"],
+                ["1/5/2017", "BEST FOR YOU ORGANICS COMPANY", "Groceries", "$27"],
+                ["1/10/2017", "COHO VINEYARD", "Restaurant", "$33"],
+                ["1/11/2017", "BELLOWS COLLEGE", "Education", "$350"],
+                ["1/15/2017", "TREY RESEARCH", "Other", "$135"],
+                ["1/15/2017", "BEST FOR YOU ORGANICS COMPANY", "Groceries", "$97"]
+            ]);  
+
+            // Queue a command to set the number format of the Amount column
+             //expensesTable.columns.getItem("AMOUNT").numberFormat = "$#";
+
+            //Autofit columns and rows
+            sheet.getUsedRange().getEntireColumn().format.autofitColumns();
+            sheet.getUsedRange().getEntireRow().format.autofitRows();
+    }
+
+    // Sort the table data
+    async function queueFilterData(sheet: Excel.Worksheet) {
+
+            // Queue a command to get the table
+            const expensesTable = sheet.tables.getItem("ExpensesTable");
+
+            // Queue a command to filter and show only certain categories
+                const filter = expensesTable.columns.getItem("CATEGORY").filter;
+                filter.apply({
+                    filterOn: Excel.FilterOn.values,
+                    values: ["Restaurant", "Groceries"]
+                });
+
+    }
+  language: typescript
+style:
+  content: ''
+  language: css
+template:
+  content: |
+    <div id="content-header" class="ms-font-m">
+        <h1>Welcome</h1>
```

Comment:
Please follow template in https://github.com/WrathOfZombies/samples/blob/master/samples/excel/range/set-get-values.yaml